cmd/operator-sdk/add/api.go

```diff
@@ -21,89 +21,262 @@ import (
 	"path/filepath"
 
 	"github.com/operator-framework/operator-sdk/cmd/operator-sdk/internal/genutil"
-	gencrd "github.com/operator-framework/operator-sdk/internal/generate/crd"
```

Comment:
Should we not add tests for it? 

I think we can scaffold a new helm type in: https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-helm.sh#L129

And for ansible in: https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible.sh

Reply:
If needed, can this scoped? @joelanford

Reply:
I would prefer to see unit tests for these changes. We can add these in a follow-up.

Reply:
I agree 100% that the unit test would be better in this case as well. The e2e is just easier fast to get done.

Reply:
Marking it as solved since anyway was defined that it could be done in a  follow-up.